### PR TITLE
[OTA-2577] Update no ungrouped devices strings

### PR DESCRIPTION
**Pull request by piotrsmol**
Signed-off-by: Piotr Smol <piotrsmol.pvt@gmail.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx [hunk 1]
```diff
@@ -88,14 +89,22 @@ class ContentPanel extends Component {
 
   render() {
     const { stores, changeFilter, showDeleteConfirmation, showEditName, addNewWizard } = this.props;
-    const { devicesStore, featuresStore, groupsStore } = stores;
+    const { featuresStore, devicesStore, groupsStore } = stores;
```
Comment (piotrcichonmee) [inline ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx]: Why changing the order?

File: ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx [hunk 2]
```diff
@@ -88,14 +89,22 @@ class ContentPanel extends Component {
...
     const { alphaPlusEnabled } = featuresStore;
+    const { 
+      devices,
```
Comment (piotrcichonmee) [inline ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx]: Please try to keep alphabetical order as possible.

File: ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx (hunk 1 above)
Comment (piotrsmol) [reply ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx]: Oops, forgot to revert this change, done :+1:

File: ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx (hunk 2 above)
Comment (piotrsmol) [reply ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx]: Done

**Pull request merged**